Dockerfile

```diff
@@ -59,6 +59,7 @@ RUN pip3 install --user bsddb3==6.2.7
 
 WORKDIR /sio2/oioioi
 
+COPY --chown=oioioi:oioioi setup.py requirements_pinned.txt ./
 COPY --chown=oioioi:oioioi setup.py requirements.txt ./
```

Comment:
setup.py is copied twice here. Imo all 3 files can be copied in one go.

Reply:
Fixed